examples/openai_embedding/example.py

```diff
@@ -0,0 +1,64 @@
+import os
```

Comment:
We might need some comments in this example code file.

Reply:
done